src/main/scala/uk/gov/hmrc/ct/ct600/v3/B586.scala

```diff
@@ -17,7 +17,17 @@
 package uk.gov.hmrc.ct.ct600.v3
 
 import uk.gov.hmrc.ct.box._
+import uk.gov.hmrc.ct.ct600.v3.calculations.B586Calculator
+import uk.gov.hmrc.ct.ct600.v3.retriever.CT600BoxRetriever
 
-case class B586(value: Option[BigDecimal]) extends CtBoxIdentifier(name = "NI Corporation Tax included") with CtOptionalBigDecimal
```

Comment:
Why do you remove Option ?

Reply:
B586 will always going to have value if the user qualifies for NIR. The value of B586 will be coming B345 if its single financial year and for the two financial year the value will be coming after adding B345 and B395 together.